api/swagger.yml

```diff
@@ -953,6 +953,11 @@ components:
         token:
           type: string
           description: opaque staging token to use to link uploaded object
+        presigned_url:
+          type: string
+          nullable: true
```

Comment:
Can we remove this one and just keet it 'non required' field? Empty have the same meaning as nil in this case and it will make the client code simple.

Reply:
Even if not required, Go still treats it as a pointer. When removing the nullable flag, the Python SDK expects a string and crashes when receiving null..